### PR TITLE
Second dataset

### PR DESCRIPTION
**Pull request by JohnFTitor**
# Hi there 👋🏻 

Thanks for taking the time to review the project

### Changes implemented in "second-dataset" branch 

- ✅  Add a new Column to the animals' table.
- ✅  Insert new data and manipulate it using transactions.
- ✅  Make queries using aggregates.

## Final notes 📔 

Is expected that this project meets the requirements. Please let me know if anything is left to be done or can be improved. Again, thanks for the review.

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,10 +1,77 @@
...
+UPDATE animals
+SET weight_kg = weight_kg * -1;
+
+ROLLBACK TO DELETE_AFTER_DATE_1;
+
+UPDATE animals
+SET weight_kg = weight_kg * -1
+WHERE weight_kg < 0;
+
+COMMIT;
```
Comment (AshakaE) [inline data.sql]: - [x] Please note, transaction queries should be in the 'queries.sql` , verify changes were made after the transaction and also verify changes persist after commit

**Review by AshakaE — CHANGES_REQUESTED** [review]
Hi @JohnFTitor  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (JohnFTitor) [reply data.sql]: Understood. Thanks!

**Review by descholar-ceo — APPROVED** [review]
Hi @JohnFTitor , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**